packages/iov-rise/src/riseconnection.ts

```diff
@@ -111,10 +117,38 @@ export class RiseConnection implements BcpConnection {
       throw new Error(`Expected one accepted transaction but got: ${JSON.stringify(response.data.accepted)}`);
     }
 
+    let blockInfoInterval: any;
+    const firstEvent: BcpBlockInfo = {
+      state: BcpTransactionState.Pending,
+    };
+    let lastEventSent: BcpBlockInfo = firstEvent;
+    const blockInfoProducer = new DefaultValueProducer<BcpBlockInfo>(firstEvent, {
+      onStarted: () => {
+        blockInfoInterval = setInterval(async () => {
+          const search = await this.searchTx({ hash: toAscii(transactionId) as TxId, tags: [] });
+          if (search.length > 0) {
+            const confirmedTransaction = search[0];
+            const event: BcpBlockInfo = {
+              state: BcpTransactionState.InBlock,
+              height: confirmedTransaction.height,
+              confirmations: confirmedTransaction.confirmations,
+            };
+
+            if (!equal(event, lastEventSent)) {
+              blockInfoProducer.update(event);
+              lastEventSent = event;
+            }
+          }
+        }, 10000);
```

Comment:
Again, please use a named constant for this.
Also a comment why it is 10s for rise and 3s for lisk (you told me different block times, but nice to add a mention in the code)

Reply:
Good point, done